### DISCUSSION TITLE
Are there plans to add video metrics in the future?

### DISCUSSION THREAD

**Discussion by Jiawei804** [Lightning-AI/torchmetrics#2799] [CompVision] [answered]
Thank you for your work. Are there plans to add video metrics in the future? Or is it a good choice to use the current image metrics to calculate single-frame video?

**Comment by Borda** [comment, marked as answer]
As of the v1.8.0 release, TorchMetrics has officially added support for video metrics, starting with VMAF (Video Multi-Method Assessment Fusion) 🎉

You can now find the full documentation and usage guide for the new VMAF metric [here](https://lightning.ai/docs/torchmetrics/stable/video/vmaf.html). It provides a perceptual video-quality score that closely mirrors human judgment, widely used by platforms like Netflix and YouTube to optimize video encoding.

Let me know if you’d like help integrating it into your workflow or comparing it with other metrics.
